docs/blog/2018-1-25-building-a-site-with-react-and-contentful/index.md

```diff
@@ -0,0 +1,208 @@
+      },
+    },
+    `gatsby-transformer-sharp`,
+    `gatsby-plugin-sharp`,
+  ],
+}
+```
+
+Once Contentful and Gatsby were getting along well together, I created pages and added a few other style-oriented plugins (Glamor and Typography) when I realized I wanted those. Then, I used GraphiQL to create queries that could pull my Contentful content into my site. After creating queries for each type of content I wanted (daily schedule, list of tricks & commands Watson knows, and a photo gallery), I saved each query in its corresponding page to see if anything broke. There were a few instances in which GraphiQL hadn’t let me know I had errors in the little ways I tried to sort data.
+
+Here’s an instance of a GraphiQL query that worked in there but not in the actual code. It needs to read `(sort: {field: [order]})` with the square brackets around the word "order" because there are multiple fields I can sort by; I had forgotten the brackets and GraphiQL said it would work, but then it didn't work. Here's the correct format:
+
+```jsx
+{
+allContentfulWatsonsEatingAndSleepSchedule (sort: {fields: [order]}) {
+edges {
+node {
+activityName
+time
+}
+}
+}
+}
+```
+
+Once I got my GraphQL queries working, I called a couple friends for help rendering the data to each page on my site. They called this “printing,” a word that I enjoyed because it reminded me of the Vincent Van Gogh poster I had. Basically, I had to create an object that could be referenced later, get stuff to appear in console so I could troubleshoot problems, and then just use the GraphQL query structure to drill down into the actual content I wanted to render on each page. Then I wrapped this all in some _light_ HTML formatting. Here's what it looks like on my site:
+
+![What the finished daily schedule looks like on my site](daily-schedule-finished.png)
+
+Following are more examples of pages I created. The code from the photo gallery page shows that I import React at the top, then I tell it to export data from Contentful by using the hierarchy in the GraphQL query. In the middle, the code shows how I’m using JS to also pull bits of data in to be rendered using the structure of the GraphQL query:
+
+```jsx
+import React from "react"
+
+export default ({ data }) => {
+  console.log(data)
+  // The next line is where the code drills deeper into the data structure to finally get to photos, at the end of the line. In order to create the correct order here, follow the structure of the GraphQL query you already created and pay attention to which fields are objects and which are arrays.
+  const photos = data.allContentfulPhotoGallery.edges[0].node.photos
+  console.log(photos)
+
+  return (
+    <div>
+      <h1>Photo Gallery</h1>
+      // The next line is where I’m map the photos data into React components.
+      {photos.map(photo => (
+        <div>
+          <br />
+          <br />
+          // The next line is where the code drills deeper into the data
```

Comment:
Same here

Reply:
ok for some reason Prettier is not respecting the line breaks when I do // within a <div>. It does fine at the top of the page.